src/duration/mod.rs

```diff
@@ -53,6 +53,9 @@ pub enum DurationParseError {
 
     #[snafu(display("failed to parse fragment value as integer"))]
     ParseIntError { source: ParseIntError },
+
+    #[snafu(display("duration overflow occured while parsing {value}{unit}"))]
```

Comment:
Can we please also include the original string in the error message?

Reply:
Included in 25e0671